python/lsst/ts/m2gui/utils.py

```diff
@@ -28,7 +28,9 @@
 from PySide2.QtGui import QPalette
 
 
-def set_button(name, callback, *args, is_checkable=False, is_indicator=False):
+def set_button(
+    name, callback, *args, is_checkable=False, is_indicator=False, is_adjust_size=False
```

Comment:
Dropping is_adjuast_size parameter shall not cause any harm, at least according to my tests. You really should not call adjustSize method yourself, Qt shall call it as needed.

Reply:
Explained above. I tried to recalculate the width or use the minimum adjusted width of button. Thanks!